srcpkgs/bolt/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'bolt'
+pkgname=bolt
+version=0.4
+revision=1
+build_style=meson
+configure_args="-Dman=true -Dsystemd=false"
+hostmakedepends="asciidoc glib-devel pkg-config"
+makedepends="elogind-devel eudev-libudev-devel libglib-devel polkit-devel"
+short_desc="Thunderbolt 3 device manager"
+maintainer="Rasmus Thomsen <rasmus.thomsen@protonmail.com>"
+license="LGPL-2.1-or-later"
+homepage="https://gitlab.freedesktop.org/bolt/bolt"
+distfiles="https://gitlab.freedesktop.org/bolt/bolt/-/archive/${version}/bolt-${version}.tar.gz"
+checksum=7692799545cd84b31aa7145712c55e36afc0d369c62157b19e5e865deae5f6f6
+
+do_check() {
+	# Tests seem to be broken
+	:
+}
+
+post_install() {
+	touch ${DESTDIR}/var/lib/boltd/.empty
```

Comment:
Discussion on irc lead to make_dirs= being the preferred way

make_dirs="/var/lib/boltd 0755 root root"